### PR TITLE
feat: add contact info

### PR DESCRIPTION
**Pull request by labs-ed**
# Description

Please include a summary of the change and which issue is fixed. Please provide the motivation for why this change is necessary at this stage of the product development cycle.

Fixes # (issue)

## Customer Impact

Please describe any customer facing impact of this change. This can be positive or negative impact.

## Performance Impact

Please describe any relevant performance impact of this change. This can be positive or negative impact. How did you characterize/test the performance impact?

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so others can reproduce. Please also list any relevant details for your test configuration.

- [x] Test A
- [x] Test B

**Hardware Configuration**:
- [ ] Dev Board
- [ ] Test rack
- [ ] DVT Boards
- [ ] Production Boards

**Software Configuration**:
* Operating System:
* Software version:
* Branch:
* Toolchain version:
* SDK version:

# Reviews

Please identify two developers to review this change

- [ ] @isnowheart 
- [ ] @matheus-rib 
- [x] @labs-ed 

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: contactInfo.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Contact Info</h1>
```
Comment (isnowheart) [inline contactInfo.html]: Adiciona um exclamação!

**Review by isnowheart — CHANGES_REQUESTED** [review]
Ajustar pontuação.

File: contactInfo.html (hunk 1 above)
Comment (labs-ed) [reply contactInfo.html]: Sinal de exclamação adicionado.

**Review by isnowheart — APPROVED** [review]
Agora tá ok!

**Pull request merged**